packages/react/popover/src/Popover.tsx

```diff
@@ -250,15 +250,13 @@ const PopoverContentImpl = forwardRefWithAs<typeof PopperPrimitive.Root, Popover
                   // as it's already setup to close, otherwise it would close and immediately open.
                   if (wasTrigger) {
                     event.preventDefault();
-                  } else {
```

Comment:
I also removed this as I don't think it makes sense, as that event is already called internally in all cases (focus outside, pointer down outside).